src/main/java/com/github/msemys/esjc/ssl/SslSettings.java

```diff
@@ -50,7 +59,20 @@ public String toString() {
      */
     public static SslSettings trustCertificateCN(String certificateCommonName) {
         checkArgument(!isNullOrEmpty(certificateCommonName), "certificateCommonName is null or empty");
-        return new SslSettings(true, certificateCommonName, true);
+        return new SslSettings(true, SslValidationMode.COMMON_NAME, certificateCommonName, null);
+    }
+
+    /**
+     * Creates a new SSL settings that enables connection encryption using SSL and
+     * trusts an X.509 server certificate which is trusted by the given certificate file.
+     *
+     * @param certificateFile server certificate file in PEM form.
+     * @return SSL settings
+     */
+    public static SslSettings trustCertificate(String certificateFile) {
```

Comment:
suggesting api change
```suggestion
    public static SslSettings trustCertificate(File certificateFile) {
```